cluster-scope/overlays/nerc-ocp-infra/acm-metrics-2ti-storage/kustomization.yaml

```diff
@@ -1,8 +1,6 @@
 apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 commonLabels:
-  app: noobaa
+  app: minio
```

Comment:
This label (both before and after the change) seems inaccurate: the resources we are deploying here aren't part of the noobaa or minio.

Reply:
absolutely right, it was an "old time" template using over and over again.
we can get rid of it.